spark/src/main/kotlin/com/adevinta/spark/components/snackbars/Snackbar.kt

```diff
@@ -93,7 +93,7 @@ public fun SparkSnackbar(
         modifier = modifier
             .padding(16.dp)
             .sparkUsageOverlay(),
-        shape = SparkTheme.shapes.extraSmall,
+        shape = SparkTheme.shapes.small,
```

Comment:
We should handle legacy:
`shape = if (LocalLegacyStyle.current) SparkTheme.shapes.extraSmall else SparkTheme.shapes.small,`